json-rpc/docs/method_submit.md

```diff
@@ -0,0 +1,80 @@
+
+## Method submit
+
+**Description**
+
+Submit a signed transaction to a full node.
+
+
+### Parameters
+
+| Name  | Type     | Description                                                                                          |
+|-------|----------|------------------------------------------------------------------------------------------------------|
+| data  | string   | Signed transaction data - hex-encoded bytes of [LCS][1] serialized Libra [SignedTransaction][3] type.|
+
+Steps to create "data" parameters:
+
+1. Create [RawTransaction][2]
+2. Create signing message:
+   1. [SHA3](https://en.wikipedia.org/wiki/SHA-3) hash bytes of string "LIBRA::RawTransaction".
+   2. [RawTransaction][2] [LCS][1] serialized bytes.
+   3. Concat above 2 bytes.
+3. Sign the message with [Ed25519](https://ed25519.cr.yp.to/) and your account private key.
+4. Create [SignedTransaction][3]
+5. Serialize [SignedTransaction][3] into bytes, and then hex-encode into string as Signed transaction data parameter.
+
+
+### Returns
+
+Null - on success
+
+Note:
+* Although submit returns errors immediatelly for the invalid transaction submitted, but success submit does not mean the transaction will be executed successfully.
+* Client should call [get_account_transaction](method_get_account_transaction.md) with the sender account address and the RawTransaction sequence_number to find out whether transaction is executed.
+* After transaction is submitted, but not executed yet, calling [get_account_transaction](method_get_account_transaction.md) will return null.
+* If [get_account_transaction](method_get_account_transaction.md) returns a Transaction, client should validate Transaction#hash == the SignedTransaction signature as it is possible there is another Transaction submitted with same account sequence number.
+* After validated the Transaction is the submitted SignedTransaction, client should confirm the transaction is executed successfully by checking [Transaction#vm_status] == "executed"; any other vm_status means execution failed. The vm_status may contain some information for client to understand what's going wrong.
```

Comment:
should say there is no partial execution in Libra, meaning the transaction either has full effect or has no effect at all.